tracing/src/test/java/com/palantir/remoting3/tracing/TracersTest.java

```diff
@@ -218,7 +218,6 @@ public void testWrapRunnableWithNewTrace_traceStateRestoredWhenThrows() throws E
 
     @Test
     public void testTraceIdGeneration() throws Exception {
-        assertThat(Tracers.randomId()).hasSize(16); // fails with p=1/16 if generated string is not padded
```

Comment:
Don’t we want to keep this?

Reply:
yep, let's keep it. functionality should be covered by the deterministic ones below, but it's good to keep  of as a piece of documentation